test/get_balance.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var Client = require('../');
+var assert = require('assert');
+var account_info = require('./fixtures/account_info')();
+var common = require('./common/common');
+var fixture = require('./fixtures/get_balance');
+
+describe('Ripple REST Client Get Balances', function() {
```

Comment:
The comment on the `build_payment` test also applies to this test and the other ones in this PR